integrations/virustotal.py

```diff
@@ -280,18 +280,19 @@ def query_api(hash: str, apikey: str) -> any:
     Exception
         If the status code is different than 200.
     """
-    params = {'apikey': apikey}
-    headers = {'x-apikey': apikey}
+    headers = {
+        "accept": "application/json",
+        'x-apikey': apikey
+    }
 
     debug('# Querying VirusTotal API')
     response = requests.get(
-        f'https://www.virustotal.com/api/v3/files/{hash}', params=params, headers=headers, timeout=timeout
+        f'https://www.virustotal.com/api/v3/files/{hash}', headers=headers, timeout=timeout    
```

Comment:
Remove trailing space

```suggestion
        f'https://www.virustotal.com/api/v3/files/{hash}', headers=headers, timeout=timeout
```